crates/vertigo-macro/src/html_parser.rs

```diff
@@ -41,6 +42,39 @@ fn convert_expresion_without_brackets(expr: Expr) -> TokenStream2 {
     expr.to_token_stream()
 }
 
+fn is_component_name(name: &str) -> bool {
+    name.chars().next().map(char::is_uppercase).unwrap_or_default()
+}
+
+fn convert_component_child(node: Node) -> TokenStream2 {
+    let cmp_stream = convert_component(node);
+    quote! {
+        .child(#cmp_stream)
+    }
+}
+
+fn convert_component(node: Node) -> TokenStream2 {
+    let component = node.name;
+    let attributes = node.attributes.into_iter()
+        .filter_map(|attr_node| {
+            if let (Some(key), Some(value)) = (attr_node.name, attr_node.value) {
+                let value = strip_brackets(value);
+                Some(quote! { #key: vertigo::to_clone(#value), })
+            } else {
+                None
+            }
+        })
```

Comment:
`
        .filter(|attr_node| {
            if let (Some(key), Some(value)) = (attr_node.name, attr_node.value) {
                let value = strip_brackets(value);
                quote! { #key: vertigo::to_clone(#value), }
            } else {
                panic!("key=value attribute expected");
            }
        })
`

Reply:
Added emit_error, instead of panic